rvic/convert.py

```diff
@@ -70,17 +72,16 @@ def uhs2param_init(config_file):
                       options['VERBOSE'])
 
     for direc in directories:
-        log.info('%s directory is %s' % (direc, directories[direc]))
+        log.info('%s directory is %s', direc, directories[direc])
```

Comment:
If you are changing these anyway, you may want to consider python type string formatting, i.e.:

```python
log.info('{} directory is {}'.format(direc, directories[direc]))
```